### PR TITLE
DEV-3 added bootstrap components in sprint-edit with inputs from spri…

### PR DESCRIPTION
**Pull request by alexsorindrago**
added bootstrap components in sprint-edit

### CODE REVIEW

**Comment by alexsorindrago** [comment]
conflicting files contain 1 modified space placed between inputs

File: src/main/resources/templates/sprint/sprint-edit.html [hunk 1]
```diff
@@ -9,32 +9,33 @@
 <div th:replace="fragments/header :: header"/>
 
 <p>Edit sprint</p>
```
Comment (cosminbucur) [inline src/main/resources/templates/sprint/sprint-edit.html:11]: in general, use `<h2>` tags for every title

File: src/main/resources/templates/sprint/sprint-edit.html [hunk 2]
```diff
@@ -9,32 +9,33 @@
...
+        <label for="dateFrom" class="form-label" id="dateFrom">Date from</label>
+        <input type="date" th:field="*{dateFrom}"/>
+    </div>
+
+    <div class="col-md-12">
+        <label for="dateTo" class="form-label" id="dateTo">Date to</label>
+        <input type="date" th:field="*{dateTo}"/>
+    </div>
+
+    <div class="col-md-12">
```
Comment (cosminbucur) [inline src/main/resources/templates/sprint/sprint-edit.html:29]: remove this input since it should not be editable

File: src/main/resources/templates/sprint/sprint-edit.html [hunk 3]
```diff
@@ -9,32 +9,33 @@
...
+        <label for="storyPoints" class="form-label">Story points</label>
+        <input type="text" class="form-control" id="storyPoints" placeholder="1 2 3 5 ..." th:field="*{storyPoints}">
+    </div>
+
+    <div class="col-12">
+        <button type="submit" class="btn btn-primary" id="save" name="save">Save</button>
+        <button type="submit" class="btn btn-secondary" id="cancel" name="cancel">Cancel</button>
```
Comment (cosminbucur) [inline src/main/resources/templates/sprint/sprint-edit.html:36]: rename just the text to **Update**. leave the id and name as it is.
put the cancel button before the save button always.

File: src/main/resources/templates/sprint/sprint-add.html [hunk 4]
```diff
@@ -35,8 +35,8 @@ <h1>Add sprint</h1>
     </div>
 
     <div class="col-12">
-        <button type="submit" class="btn btn-secondary" id="cancel" name="cancel">Cancel</button>
         <button type="submit" class="btn btn-primary" id="save" name="save">Save</button>
+        <button type="submit" class="btn btn-secondary" id="cancel" name="cancel">Cancel</button>
```
Comment (cosminbucur) [inline src/main/resources/templates/sprint/sprint-add.html:39]: revert this change. cancel button should be first. and the action button should be second.
also, avoid modifying files outside of your task scope. stick to edit sprint task.

**Review by cosminbucur — CHANGES_REQUESTED** [review]

**Review by cosminbucur — APPROVED** [review]
